languages/php/src/BitwardenClient.php

```diff
@@ -3,11 +3,38 @@
 namespace Bitwarden\Sdk;
 
 use Bitwarden\Sdk\Schemas\AccessTokenLoginRequest;
-use Bitwarden\Sdk\schemas\ClientSettings;
+use Bitwarden\Sdk\Schemas\ClientSettings;
 use Bitwarden\Sdk\Schemas\Command;
+use Bitwarden\Sdk\Schemas;
 use FFI;
-use Swaggest\JsonDiff\Exception;
 
+class AuthClient {
+    private CommandRunner $commandRunner;
+
+    public function __construct(CommandRunner $commandRunner) {
+        $this->commandRunner = $commandRunner;
+    }
+
+    /**
+     * @throws \Exception
+     */
+    public function login_access_token(string $access_token, string $state_file): void
+    {
+        $access_token_request = new AccessTokenLoginRequest();
+        $access_token_request->accessToken = $access_token;
+        $access_token_request->stateFile = $state_file;
+        $command = new Command();
+        $command->loginAccessToken = $access_token_request->jsonSerialize();
+        $result = $this->commandRunner->run($command);
+        if (!isset($result->authenticated)) {
+            throw new \Exception("Authorization error");
+        }
+
+        if ($result->authenticated == False) {
+            throw new \Exception("Unauthorized");
+        }
```

Comment:
It might be useful to know why there was an error. Also from what i observed, it is impossible to get into scenario where `authenticated` is false for SM - but we should leave it just in case anyway.

```suggestion
        try {
            $result = $this->commandRunner->run($command);
            if (!isset($result->authenticated) || !$result->authenticated) {
                throw new Exception("Unauthorized");
            }
        } catch (Exception $exception) {
            throw new Exception("Authorization error: " . $exception->getMessage());
        }
```